package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
So, the workflow for dev would look like this:
1. (once) clone halfpenny to a separate repo on disk and run `npm link`
1. checkout a feature branch of halfpenny and of the nodeapi
1. perform dev work in both halfpenny and nodeapi
1. once it's ready to go, modify increment halfpenny's version number in halfpenny/package.json AND in nodeapi/package.json
1. during code review, the reviewer will also need to `npm link` and checkout the feature branch of halfpenny (since it won't be published to NPM until after merging).
1. After code review and merge, publish the new version of halfpenny to NPM

Is that the way that you envisioned it as well?

The only potential issue that I see is conflicting version number changes if multiple features are being developed in parallel. For example:

Chris and I are both working on separate nodeapi features. We both bump the halfpenny version from 3.0.1 to 3.1.0. Chris' feature gets merged right away (because he's awesome), but my feature takes a while. After Chris' feature is merged, but before my feature is merged, Chris merges another feature that bumps to 3.2.0. If my PR is merged without conflicts, it will set the version number back down to 3.1.0 in both nodeapi and halfpenny. This scenario may result in a conflict, but I'm not 100% sure.